src/typescript/frontend/package.json

```
@@ -0,0 +1,78 @@
+{
+  "name": "econia-labs",
+  "main": "src/index.tsx",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "@popperjs/core": "^2.11.8",
+    "@reduxjs/toolkit": "^1.9.5",
+    "axios": "^1.5.0",
+    "formik": "^2.4.4",
+    "framer-motion": "^10.16.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-helmet-async": "^1.3.0",
+    "react-popper": "^2.3.0",
+    "react-redux": "^8.1.2",
+    "react-router-dom": "^6.15.0",
+    "react-scripts": "5.0.1",
+    "react-toastify": "^9.1.3",
+    "redux-logger": "^3.0.6",
+    "socket.io-client": "^4.7.2",
+    "styled-components": "^6.1.8",
+    "styled-system": "^5.1.5",
+    "stylis": "^4.3.1",
+    "web-vitals": "^2.1.4",
+    "yup": "^1.2.0"
+  },
+  "scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "build-dev": "chmod +x ./scripts/build-dev.sh && ./scripts/build-dev.sh",
+    "build-prod": "chmod +x ./scripts/build-prod.sh && ./scripts/build-prod.sh",
+    "test": "react-app-rewired test",
+    "eject": "react-app-rewired eject",
+    "lint": "eslint -c .eslintrc.json --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "npm run lint -- --fix",
+    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc.js",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "prepare": "husky install",
```

```suggestion
    "prepare": "cd ../../../ && husky install",
```

This is how I got it to install/build, is this the proper way to do it? I also had to run `npm install --force`. Let me know if there's a better way to build